spec/unit/source/git_spec.rb

```diff
@@ -122,7 +130,11 @@
   describe "generate_environments respects filter_command setting and name" do
     before do
       allow(subject.cache).to receive(:branches).and_return ['master', 'development', 'production']
-      subject.instance_variable_set(:@filter_command, '[ $R10K_NAME = mysource ]')
+      if R10K::Util::Platform.windows?
+        subject.instance_variable_set(:@filter_command, 'powershell.exe if ($env:R10K_BRANCH.equals(\"mysource\")) {exit 1} else {exit 0}')
```

Comment:
I still think these exit codes might need to be switched? In the other similar non-windows tests, we're checking for `$R10K_BRANCH != <branch>` but this one is checking for equality.

Reply:
oh, the issue was with $R10K_NAME vs $R10K_BRANCH. But you are also right, it should be switched as well.